src/main/java/Bulldozer.java

```diff
@@ -0,0 +1,9 @@
+class Bulldozer extends Machine {
+    public void doWork() {
+        System.out.println("Truck started working.");
+    }
+
+    public void stopWork() {
+        System.out.println("Truck stopped working.");
```

Comment:
Similarly, the stop message should also reflect that it is a `Bulldozer` that stopped working. Please correct the class name in the output message.